<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
    <item>
     <layout class="QHBoxLayout" name="layoutAtTop" stretch="0,1,0,1,1,1,20,1,1,1,1,1">
      <item>
       <spacer name="hSpace1">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonInfo">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; text-decoration: underline;&quot;&gt;KEYBOARD CONTROLS:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[p]&lt;/span&gt; open profile menu&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt; (add/delete profile &amp;amp; edit profile names)&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[‚Üë ‚Üì ‚Üê ‚Üí]&lt;/span&gt; move button selection cursor&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[c]&lt;/span&gt; copy content into clipboard&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[v]&lt;/span&gt; paste content from clipboard&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[s] &lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;outside of search:&lt;/span&gt; focus text field &lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;(enter/return: search, +shift/ctrl: add)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[s] &lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;during search:&lt;/span&gt; reset current search &amp;amp; focus text field&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[+]&lt;/span&gt; add a new &lt;span style=&quot; font-size:7pt;&quot;&gt;(empty)&lt;/span&gt; button to the grid&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[-]&lt;/span&gt; delete ALL buttons that are currently marked for deletion &lt;span style=&quot; font-size:8pt; font-style:italic; color:#ff0000;&quot;&gt;(red)&lt;/span&gt;&lt;span style=&quot; font-style:italic; color:#ff0000;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;(if no buttons have been marked for deletion then &lt;/span&gt;-&lt;span style=&quot; font-size:7pt;&quot;&gt; works like backspace/delete)&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[m] &lt;/span&gt;Move currently selected button and change its position in the grid&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[CTRL/Shift &amp;amp; -]&lt;/span&gt; (un)mark the currently selected button for deletion&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[Backspace/Delete]&lt;/span&gt; delete the currently selected button&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[CTRL/Shift &amp;amp; Backspace/Delete]&lt;/span&gt; same as &lt;span style=&quot; font-style:italic;&quot;&gt;CTRL/Shift &amp;amp; -&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[Return/Enter]&lt;/span&gt; open edit dialog for currently selected button&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[CTRL/Shift &amp;amp; Enter/Return]&lt;/span&gt; open context / right-click menu&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt; (add title, ...)&lt;br/&gt;&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;[ESC]&lt;/span&gt; reset button selection &amp;amp; all deletion-marks &lt;span style=&quot; font-size:8pt; font-style:italic; color:#ff0000;&quot;&gt;(red buttons)&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-style:italic; color:#33d17a;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#26a269;&quot;&gt;Most options are also available via &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; text-decoration: underline; color:#26a269;&quot;&gt;right-clicking&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#26a269;&quot;&gt; on a button.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { border: none; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::DialogInformation"/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpace2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="profileLabel">
        <property name="text">
         <string>Current profile:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonProfile">
        <property name="text">
         <string>&lt;create here&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpace3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="textInputField">
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Insert text or URL, then click on üîç or Add</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpace4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSearch">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::SystemSearch"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpace5">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpace6">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>340</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="scrollGrid"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
